### PR TITLE
filemenu

### PR DESCRIPTION
**Pull request by Unnunu**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀 This PR matches 3 functions (+0.06%) on `us`.

✅ This PR fixes 6 warnings!

File: src/filemenu/filemenu_createfile.c [hunk 1]
```diff
@@ -0,0 +1,485 @@
...
+                            : SCREEN_WIDTH / 2) - gWindows[WINDOW_ID_FILEMENU_YESNO_PROMPT].width / 2;
+
+        newMenu2 = filemenu_menus[1];
+        filemenu_currentMenu = 1;
+        newMenu2->page = 2;
+        filemenu_set_selected(newMenu2, 0, 0);
+    }
+}
+
+static const f32 padding[2] = { 0.0f, 0.0f }; // not sure why this is needed
```
Comment (nanaian) [inline src/filemenu/filemenu_createfile.c:474]: File split?

**Review by nanaian — APPROVED** [review]
amazing

File: src/filemenu/filemenu_createfile.c (hunk 1 above)
Comment (Unnunu) [reply src/filemenu/filemenu_createfile.c:474]: If I remove this line, rodata section doesn't change at all, but `filemenu_RODATA_END` becomes wrong, which is referenced in `state_file_select.c`. So I'm not sure if splitting can fix this.

**Pull request merged**